test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -122,6 +122,19 @@ public void AutoFitRegressionTest()
 
         [LightGBMFact]
         public void AutoFitRankingTest()
+        {
+            // Testing Ranking with both TrainValidateRunner and CrossValSummaryRunner
+            // with corresponding baseline metrics. The metrics of Ranking with
+            // CrossValSummaryRunner is lower due to the 10 cross validation folds
+            // and due to the fact that CrossValSummaryRunner is used only when
+            // fewer than <1500 rows of training data is given.
+            AutoFitRankingTestTemplate(true, 0.4, 20);
+            AutoFitRankingTestTemplate(false, 0.4, 28);
+            AutoFitRankingCVTestTemplate(true, 0.4, 19);
+            AutoFitRankingCVTestTemplate(false, 0.31, 15);
+        }
+
+        private void AutoFitRankingTestTemplate(bool isUsingTrainValidateRunner, double minExpectedNdcg, double minExpectedDcg)
```

Comment:
The function name and what it does is confusing. My guess is this template should only run train/validation ranking experiment because there's another function (AutoFitRankingCVTestTemplate) which looks like should only run cross-validation training.

Reply:
Agreed, thank you.